packages/plugin-hardhat/src/deploy-proxy.ts

```diff
@@ -71,25 +78,29 @@ export function makeDeployProxy(hre: HardhatRuntimeEnvironment): DeployFunction
     inst.deployTransaction = proxyDeployment.deployTransaction;
     return inst;
   };
+}
 
-  function getInitializerData(ImplFactory: ContractFactory, args: unknown[], initializer?: string | false): string {
-    if (initializer === false) {
-      return '0x';
-    }
+export function getInitializerData(
```

Comment:
Can we move this to a file in utils, now that it's also used by `deploy-beacon-proxy`?